src/number_theory/divisors.lean

```diff
@@ -91,6 +91,8 @@ begin
     apply nat.le_of_dvd (nat.succ_pos m) hdvd }
 end
 
+lemma mem_divisors_self (n : ℕ) (h : n ≠ 0) : n ∈ n.divisors := by simpa
```

Comment:
I'd lean towards the term mode proof here, since it makes it obvious that this is proved via the lemma above it.
```suggestion
lemma mem_divisors_self (n : ℕ) (h : n ≠ 0) : n ∈ n.divisors := mem_divisors.2 ⟨dvd_rfl, h⟩
```